catalog/spatial/registry/registry-ebrim-transformer/src/main/java/org/codice/ddf/registry/converter/RegistryPackageConverter.java

```diff
@@ -490,24 +493,39 @@ private static void setMetacardEmailAttribute(List<EmailAddressType> emailAddres
 
     private static void setMetacardPhoneNumberAttribute(List<TelephoneNumberType> phoneNumbers,
             String metacardAttribute, MetacardImpl metacard) {
+
         List<String> metacardPhoneNumbers = new ArrayList<>();
+        if (CollectionUtils.isNotEmpty(phoneNumbers)) {
+            for (TelephoneNumberType digits : phoneNumbers) {
+                metacardPhoneNumbers.add(getPhoneNumber(digits));
 
-        for (TelephoneNumberType digits : phoneNumbers) {
+                if (CollectionUtils.isNotEmpty(metacardPhoneNumbers)) {
+                    metacard.setAttribute(metacardAttribute, (Serializable) metacardPhoneNumbers);
+                }
+            }
+        }
+    }
 
-            if (StringUtils.isBlank(digits.getExtension())) {
-                metacardPhoneNumbers.add(String.format("(%s) %s",
-                        digits.getAreaCode(),
-                        digits.getNumber()));
+    private static StringBuilder buildNonNullString(StringBuilder stringBuilder,
+            String attributeValue, String format, String delimiterBefore) {
+        if (StringUtils.isNotBlank(attributeValue)) {
+            if (stringBuilder.length() > 0) {
+                stringBuilder.append(delimiterBefore);
+            }
+            if (StringUtils.isNotEmpty(format)) {
+                stringBuilder.append(String.format(format, attributeValue));
             } else {
-                metacardPhoneNumbers.add(String.format("(%s) %s extension %s",
-                        digits.getAreaCode(),
-                        digits.getNumber(),
-                        digits.getExtension()));
+                stringBuilder.append(attributeValue);
             }
         }
-        if (CollectionUtils.isNotEmpty(metacardPhoneNumbers)) {
-            metacard.setAttribute(metacardAttribute, (Serializable) metacardPhoneNumbers);
-        }
+        return stringBuilder;
+    }
+
+    private static StringBuilder buildNonNullString(StringBuilder stringBuilder,
+            String attributeValue, String delimiterBefore) {
+
+        return (buildNonNullString(stringBuilder, attributeValue, "", delimiterBefore));
```

Comment:
Seems weird to me in a private method not to use `null` as your "I'm not here" flag.